src/processor/reconnection-graph.service.ts

```diff
@@ -49,23 +56,31 @@ export class ReconnectionGraphService implements OnApplicationBootstrap, OnAppli
 
     // TODO set state based on the response from getUserGraphFromProvider
     const [graphConnections, graphKeyPair] = await this.getUserGraphFromProvider(dsnpUserId, providerId);
-    this.logger.log('graphConnections', graphConnections);
-    this.logger.log('graphKeyPair', graphKeyPair);
 
     // graph config and respective schema ids
-    const graphSdkConfig = await this.graphStateManager.getGraphConfig();
-    const public_follow_schema_id = await this.graphStateManager.getSchemaIdFromConfig(ConnectionType.Follow, PrivacyType.Public);
-    const public_friendship_schema_id = await this.graphStateManager.getSchemaIdFromConfig(ConnectionType.Friendship, PrivacyType.Public);
-    const private_follow_schema_id = await this.graphStateManager.getSchemaIdFromConfig(ConnectionType.Follow, PrivacyType.Private);
-    const private_friendship_schema_id = await this.graphStateManager.getSchemaIdFromConfig(ConnectionType.Friendship, PrivacyType.Private);
+    const graphSdkConfig  = await this.graphStateManager.getGraphConfig();
+    
+    // get the user's DSNP Graph from the blockchain and form import bundles
+    const importBundles = await this.formImportBundles(dsnpUserId, graphSdkConfig, graphKeyPair);
+    await this.graphStateManager.importUserData(importBundles).then((results) => {
+      throw new Error(JSON.stringify("importUserData results: " + results));
+    });
```

Comment:
Don't mix & match `async/await` with `Promise.then()/.catch()`.

In an `async` function, do:
```
try {
   await function-returning-Promise
   // next, do whatever you would in Promise.then()
} catch (err) {
   // do whatever you would in Promise.catch()
}
```

non-async method:
```
functionReturningPromise()
   .then(() => /* success processing */)
   .catch((reason) => /* error processing*/)
```